Content.Shared/Clothing/Components/EmitsSoundOnMoveComponent.cs

```diff
@@ -0,0 +1,35 @@
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using Robust.Shared.Map;
+
+namespace Content.Shared.Clothing.Components;
+
+/// <summary>
+///   Indicates that the clothing entity emits sound when it moves.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class EmitsSoundOnMoveComponent : Component
```

Comment:
This needs to go into a different PR

Reply:
This is needed for this PR, i am not making a new PR just for for this system.

Reply:
Keep this, then move wagging into a different PR.